test/oli/sections_test.exs

```diff
@@ -338,6 +338,65 @@ defmodule Oli.SectionsTest do
       remixed_projects = Sections.get_remixed_projects(section.id, section.base_project_id)
       assert 2 = length(remixed_projects)
     end
+
+    test "get_active_sections_by_project/2 returns course sections actives for a project" do
```

Comment:
Maybe we can add a test for the cases where the two new functions in the context returns empty (or zero). Wdyt?